storage/storage_image.go

```diff
@@ -341,16 +355,31 @@ func (s *storageImageDestination) Commit() error {
 	}
 	logrus.Debugf("created new image ID %q", img.ID)
 	s.ID = img.ID
+	names := []string{}
+	if s.imageRef.DockerReference() != nil {
+		names = append(names, verboseName(s.imageRef.DockerReference()))
+	}
 	if s.Tag != "" {
```

Comment:
Yeah, you're right, we don't need to add Tag as a name.  That simplifies things.